apis/openstack-swift/src/main/java/org/jclouds/openstack/swift/v1/blobstore/RegionScopedSwiftBlobStore.java

```diff
@@ -819,10 +817,7 @@ public void run() {
                }
             }
             // Finished writing results to stream
-            try {
-               output.close();
-            } catch (IOException e) {
-            }
+            Closeables2.closeQuietly(output);
```

Comment:
> enclose the entire for block in a try/finally block

I merged the fixes for now, I will see if I can do a somewhat more complex refactoring in a separate PR later. I wanted to spend some time looking at reusing this code for BaseBlobstore first, before I spend too much time on it.